uk.co.compucorp.civicrm.hrleaveandabsences/js/hrleaveandabsences.form.manage_entitlements.js

```diff
@@ -29,6 +30,46 @@ CRM.HRLeaveAndAbsencesApp.Form.ManageEntitlements = (function($) {
     });
   };
 
+  /**
+   * Add event listeners to events triggered by elements of managed by this class
+   *
+   * @private
+   */
+  ManageEntitlements.prototype._addEventListeners = function() {
+    this._listElement.find('tbody > tr').on('click', this._onListRowClick.bind(this));
+  };
+
+  /**
+   * This is the event handler for when the user clicks on a row of the calculations
+   * list.
+   *
+   * It shows the user a popup with details of the selected calculation. Even if the
+   * proposed entitlement was overridden, we display the original calculation.
+   *
+   * @param event
+   * @private
+   */
```

Comment:
Actually this method is still public - you can access it from outside.

As long as this is closure you can make this method fully private by using normal function statement:
````javascript
function addEventListeners() {}
````

This way it won't be accessible outside class.